client/app/pages/dashboards/hooks/utils.js

```diff
@@ -0,0 +1,20 @@
+export function getAllowedWidgetsForCurrentParam(dashboardParameters, dashboardAllowedWidgets, dashboardAllWidgets) {
+  try {
+    // filter the widgets by keeping only the ones we want
+    for (var param of dashboardParameters) {
+      const paramValue = param.value;
```

Comment:
I'm not sure if it's a good idea to filter using any parameter. This can create some inconsistencies.  I prefer to have a hard-coded name for the parameter we will use.

What do you think @helenalebreton?

Right now if a parameter value matches any row in the allowed_widget it will be used.

Reply:
@thiagogds do you mean the main parameter ? I am not sure I follow - maybe a quick call on that one.

Reply:
or do you mean the fact that if one value from the widget field in the allowed_widgets query matches at least one tag of a widget (currently simulated with visualisation name), then we show the widget ?